docs/db/changelog/changesets/enrollment-server-onboarding/1.6.x/20230530-add-column-total-attempts.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <changeSet id="1" logicalFilePath="enrollment-server/1.6.x/20230530-add-column-total-attempts.xml" author="Jan Dusil">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="es_onboarding_otp" columnName="total_attempts"/>
+            </not>
+        </preConditions>
+        <comment>Add total_attempts column</comment>
+        <addColumn tableName="es_onboarding_otp">
+            <column name="total_attempts" type="integer" />
```

Comment:
What will be the default value? Are we ok with `null`? The application is already deployed in production.